src/scriptengine/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Script Engine is a base for scripting in OpenStudio
+# See implementations in ruby/engine and python/engine
+
+add_library(scriptengine INTERFACE)
+target_include_directories(scriptengine INTERFACE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I'm confused by this subdirectory, and it's location at src/scriptengine. Right now it's only used by the ruby/engine target.

Reply:
Soon Python will use it also.